examples/plot_missing_values.py

```diff
@@ -1,73 +1,107 @@
 """
-======================================================
+====================================================
 Imputing missing values before building an estimator
-======================================================
-
-This example shows that imputing the missing values can give better
-results than discarding the samples containing any missing value.
-Imputing does not always improve the predictions, so please check via
-cross-validation.  Sometimes dropping rows or using marker values is
-more effective.
+====================================================
 
 Missing values can be replaced by the mean, the median or the most frequent
```

Comment:
What do you think of just dropping these paragraphs? The example's job is to show it in action not to explain the available usages.

Reply:
It looks like it's already gone when I look here: https://github.com/scikit-learn/scikit-learn/pull/8478/files